packages/components/src/popover/README.md

```diff
@@ -49,130 +51,133 @@ render(
 
 The component accepts the following props. Props not included in this set will be applied to the element wrapping Popover content.
 
-### focusOnMount
-
-By default, the _first tabblable element_ in the popover will receive focus when it mounts. This is the same as setting `focusOnMount` to `"firstElement"`. If you want to focus the container instead, you can set `focusOnMount` to `"container"`.
+### `anchorRect`: `DomRectWithOwnerDocument`
 
-Set this prop to `false` to disable focus changing entirely. This should only be set when an appropriately accessible substitute behavior exists.
+An object extending a `DOMRect` with an additional optional `ownerDocument` property, used to specify a fixed popover position.
 
--   Type: `String` or `Boolean`
 -   Required: No
--   Default: `"firstElement"`
 
-### placement
+### `anchorRef`: `Element | PopoverAnchorRefReference | PopoverAnchorRefTopBottom | Range`
+
+Used to specify a fixed popover position. It can be an `Element`, a React reference to an `element`, an object with a `top` and a `bottom` properties (both pointing to elements), or a `range`.
 
-The direction in which the popover should open relative to its parent node or anchor node.
+-   Required: No
 
-The available base placements are 'top', 'right', 'bottom', 'left'.
+### `animate`: `boolean`
 
-Each of these base placements has an alignment in the form -start and -end. For example, 'right-start', or 'bottom-end'. These allow you to align the tooltip to the edges of the button, rather than centering it.
+Whether the popover should animate when opening.
 
--   Type: `String`
 -   Required: No
--   Default: `"bottom-start"`
+-   Default: `true`
 
-### flip
+### `children`: `ReactNode`
 
-Specifies whether the `Popover` should flip across its axis if there isn't space for it in the normal placement.
+The `children` elements rendered as the popover's content.
 
-When the using a 'top' placement, the `Popover` will switch to a 'bottom' placement. When using a 'left' placement, the popover will switch to a 'right' placement.
+-   Required: Yes
 
-The `Popover` will retain its alignment of 'start' or 'end' when flipping.
+### `expandOnMobile`: `boolean`
+
+Show the popover fullscreen on mobile viewports.
 
--   Type: `Boolean`
 -   Required: No
--   Default: `true`
 
-### resize
+### `flip`: `boolean`
+
+Specifies whether the popover should flip across its axis if there isn't space for it in the normal placement.
+
+When the using a 'top' placement, the popover will switch to a 'bottom' placement. When using a 'left' placement, the popover will switch to a `right' placement.
 
-Adjusts the height of the `Popover` to prevent overflow.
+The popover will retain its alignment of 'start' or 'end' when flipping.
 
--   Type: `Boolean`
 -   Required: No
 -   Default: `true`
 
-### shift
+### `focusOnMount`: `'firstElement' | boolean`
```

Comment:
Removed `container` as a possible option as it looks like it was removed in https://github.com/WordPress/gutenberg/pull/27699